### PR TITLE
Add module level logger.

### PR DESCRIPTION
**Pull request by lee12jin**
@jsm set up this logger

### CODE REVIEW

File: pistachio/__init__.py [hunk 1]
```diff
@@ -1 +1,6 @@
+import logging
+
 from .main import *
+
+if __name__ == '__main__':
+  logging.basicConfig(format='%(levelname)s:Pistachio: %(message)s', level=logging.INFO)
```
Comment (jsm) [inline pistachio/__init__.py]: Add a newline

File: pistachio/s3.py [hunk 2]
```diff
@@ -99,9 +101,9 @@ def fetch_config_partial(folder, key):
     config_partials[folder].append(yaml.load(contents))
 
   except botocore.exceptions.ClientError as e:
-    logging.warning("[Pistachio] S3 exception on %s: %s" % (key, e))
+    logger.warning("[Pistachio] S3 exception on %s: %s" % (key, e))
```
Comment (jsm) [inline pistachio/s3.py]: Need to remove [Pistachio] here and below

**Comment by jsm** [comment]
LGTM

**Comment by lee12jin** [comment]
:D

**Pull request merged**